internal/cache/serialization.go

```diff
@@ -62,12 +63,18 @@ func (c *Cache) dumpToYaml() (string, error) {
 	c.mu.RLock()
 	defer c.mu.RUnlock()
 
+	username := "root"
+	currentUser, _ := user.Current()
```

Comment:
I think it's still good to catch and return the error here and let the test deal with it.